receiver/snowflakereceiver/documentation.md

```diff
@@ -0,0 +1,84 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# snowflakereceiver
+
+## Metrics
+
+These are the metrics available for this scraper.
+
+| Name | Description | Unit | Type | Attributes |
+| ---- | ----------- | ---- | ---- | ---------- |
+| **snowflake.billing.cloud_service.sum** | cloud services sum | credits | Gauge(Double) | <ul> <li>service_type</li> </ul> |
+| **snowflake.billing.total_credit.sum** | billing total sum | credits | Gauge(Double) | <ul> <li>service_type</li> </ul> |
+| **snowflake.billing.virtual_warehouse.sum** | compute credits used sum |  | Gauge(Double) | <ul> <li>service_type</li> </ul> |
+| **snowflake.billing.warehouse.cloud_service.sum** | ware billing for cloud service | credits | Gauge(Double) | <ul> <li>warehouse_name</li> </ul> |
```

Comment:
Is this metrics based on the same data as snowflake.billing.cloud_service.sum just broken down by different attribute? Can we use just `snowflake.billing.cloud_service.sum` with both `warehouse_name` and `service_type` attributes?

Reply:
No so all of the data represented in each metric is actually the result of a query run in snowflake (this is how they do reporting) so they're all actually distinct values.